frontend/src/services/ApiService.ts

```diff
@@ -43,8 +43,10 @@ export const transformAxiosError = (error: Error): ApiError => {
         meta: {
           action: 'transformAxiosError',
           error: {
+            code: error?.code,
             message: error?.message,
             stack: error?.stack,
+            dump: error?.toJSON(),
```

Comment:
Hmm, I don't think errors are guaranteed to have a `.toJSON()` function associated with them, so this could cause a runtime error. Could you double check?

Reply:
Quick test in chrome console:
![image](https://user-images.githubusercontent.com/935223/228405967-b16c8f41-9676-49b4-b2ea-a37fab681f82.png)

I've pushed a commit to add a safety check.

Reply:
Yes indeed! That is the reason why we read properties explicitly (`message`, `stack`) to add them to the logs. It's a bit sad, but manageable when we know what to do.